apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/S3MinioConfig.java

```diff
@@ -0,0 +1,32 @@
+package gov.cms.bfd.pipeline.ccw.rif.extract.s3;
+
+/**
+ * S3MinioConfig is an object for configuring minio to use as a S3 bucket for local development
+ * Username, password, and minioaddress can be passed in as maven parameters
+ */
+public class S3MinioConfig {
+
+  public String minioUserName;
+  public String minioPassword;
+  public String minioEndpointAddress;
+  public Boolean useMinio;
+
+  private static S3MinioConfig single_instance = null;
+
+  private S3MinioConfig() {
+    minioUserName = System.getProperty("s3.localUser", "bfdLocalS3Dev");
+    minioPassword = System.getProperty("s3.localPass", "bfdLocalS3Dev");
+    minioEndpointAddress = System.getProperty("s3.localAddress", "http://localhost:9000");
+    useMinio = Boolean.parseBoolean(System.getProperty("s3.local"));
```

Comment:
This comment was on a previous version before your moved this but you can simplify this to `Boolean.getBoolean("s3.local")` which will look up the system property and do the right things without having to actually get the system property and parse it yourself.